PublicMapping/districtbuilder
# Feature/mvm/block warning modal

## Overview

Adds a warning modal to add some friction before utilizing the block workflow for the first time on a given project

### Checklist


### Demo

![localhost_3003_projects_d802c294-034b-4d20-96d7-0cd1ed3a7e95](https://user-images.githubusercontent.com/4432106/91920197-55fb3f00-ec96-11ea-9846-e821ae337b6a.png)


### Notes

 - @tgilcrest The copy on the modal may need some adjustment - the wireframes still had some lorem ipsum.
 - This PR is based on top of #371 to avoid merge conflicts. I'll rebase on top of `develop` and change which branch this PR is pointing to after that PR is merged.

## Testing Instructions

- `scripts/update`
- Attempt to enter block-level editing mode. You should now instead first see the warning modal.

Closes #284
